src/tests/JIT/HardwareIntrinsics/Arm/Shared/Helpers.cs

```diff
@@ -4136,6 +4136,32 @@ public static byte AddSaturate(byte op1, byte op2)
             return ovf ? byte.MaxValue : result;
         }
 
+        public static double[] AddSequentialAcross(double[] op1, double[] op2)
+        {
+            double[] result = new double[op1.Length];
+            result[0] = op1[0];
+
+            foreach (double num in op2)
```

Comment:
This only calculates the sum for active lanes, so you need to check that as well.

Reply:
Got it, fixed.